go.mod

```diff
@@ -1,7 +1,12 @@
 module github.com/candid82/joker
 
 require (
+	github.com/chzyer/logex v1.1.10 // indirect
```

Comment:
Not sure where this came from. I just upgraded go to 1.13 and rebuilt Joker, and the only thing that changed was `go 1.13` line at the bottom of this file (https://github.com/candid82/joker/commit/dc5b05857daf2fdbf55cf465374febd60a8be444).

Reply:
> Not sure where this came from. I just upgraded go to 1.13 and rebuilt Joker, and the only thing that changed was `go 1.13` line at the bottom of this file ([dc5b058](https://github.com/candid82/joker/commit/dc5b05857daf2fdbf55cf465374febd60a8be444)).

Color me befuddled! The `go 1.13` line is all I've noticed being added lately, when building canonical Joker on my go1.13 dev machines.